release notes/v0.48.0.md

```diff
@@ -0,0 +1,257 @@
+k6 v0.48.0 is here 🎉! This release includes:
+
+- Remote modules should be specified with full URLs.
+- Metrics names now need to conform to both OpenTelemetry (OTEL) and Prometheus name requirements.
+- A new `k6 new` subcommand to generate a new test script.
+- A new `k6/experimental/fs` module for file interactions.
+- k6 browser got a support for CPU and network throttling.
```

Comment:
I don't think the first line is all that relevant - I have not seen this being done in the wild ... forever. So likely 99% of users will not even understand what has been removed.

Maybe we can just say "a bunch of breaking changes that have been waiting for a while, see the list below." ?